src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -235,6 +227,293 @@ object NguhBlocks {
             .mapColor(MapColor.STONE_GRAY)
     )
 
+    // Cinnabar blocks
```

Comment:
```suggestion
    // =========================================================================
    //  Cinnabar Blocks
    // =========================================================================
```
This is a minor thing, but please do this (and so on for the other sections here) so it’s easier to figure out how things are grouped together